pubnub/endpoints/access/grant_token.py

```diff
@@ -32,6 +32,18 @@ def authorized_uuid(self, uuid):
         self._authorized_uuid = uuid
         return self
 
+    def authorized_user(self, user):
+        self._authorized_uuid = user
+        return self
+
+    def spaces(self, spaces):
+        self._channels = spaces
+        return self
+
+    def users(self, users):
+        self._uuids = users
+        return self
+
     def channels(self, channels):
```

Comment:
Do we want to deprecate it? Don't know how to do it in python :D

Reply:
yeah i would but i'm not sure when it will happen